tests/test_func.py

```diff
@@ -94,46 +94,34 @@ def test_func_on_more_than_one_table(db: gp.Database):
 
 def test_simple_agg(db: gp.Database):
     rows = [(i,) for i in range(10)]
-    numbers = gp.values(rows, db=db, column_names=["val"])
+    numbers = gp.to_table(rows, db=db, column_names=["val"])
     count = gp.aggregate_function("count")
 
-    results = list(count(numbers["val"], db=db).to_table().fetch())
+    results = list(numbers.group_by().assign(count=lambda t: count(t["val"])).fetch())
     assert len(results) == 1 and results[0]["count"] == 10
 
 
 def test_agg_group_by(db: gp.Database):
     rows = [(i, i % 2 == 0) for i in range(10)]
-    numbers = gp.values(rows, db=db, column_names=["val", "is_even"])
+    numbers = gp.to_table(rows, db=db, column_names=["val", "is_even"])
     count = gp.aggregate_function("count")
 
     # FIXME: Remove extraneous rename() in group_by() after spearating Expr
     # with NamedExpr.
-    results = list(
-        count(
-            numbers["val"],
-            group_by=numbers.group_by("is_even"),
-            db=db,
-        )
-        .to_table()
-        .fetch()
-    )
+    results = numbers.group_by("is_even").assign(count=lambda t: count(t["val"])).fetch()
     assert len(results) == 2
     for row in results:
         assert ("is_even" in row) and (row["is_even"] is not None) and (row["count"] == 5)
 
 
 def test_agg_group_by_multi_columns(db: gp.Database):
     rows = [(i, i % 2 == 0, i % 3 == 0) for i in range(6)]  # 0, 1, 2, 3, 4, 5
-    numbers = gp.values(rows, db=db, column_names=["val", "is_even", "is_multiple_of_3"])
+    numbers = gp.to_table(rows, db=db, column_names=["val", "is_even", "is_multiple_of_3"])
     count = gp.aggregate_function("count")
 
-    results = list(
-        count(
-            numbers["val"],
-            group_by=numbers.group_by("is_even", "is_multiple_of_3"),
-            db=db,
-        )
-        .to_table()
+    results = (
+        numbers.group_by("is_even", "is_multiple_of_3")
```

Comment:
So for **group_by**, we will only allow the syntax `tab.group_by.assign()`, no more `tab.assign(lambda t: func(t, group_by=t.group_by())`? (I am totally ok with this modification)